src/app/tests/suites/certification/Test_TC_CC_10_1.yaml

```diff
@@ -0,0 +1,1041 @@
+                value: 0x00
+              - name: "GroupID"
+                value: G1
+              - name: "SceneList"
+                value: []
+
+    - label: "Step 1d: TH reads ColorTempPhysicalMinMireds attribute from DUT"
+      PICS: CC.S.A400b
+      cluster: "Color Control"
+      command: "readAttribute"
+      attribute: "ColorTempPhysicalMinMireds"
+      response:
+          saveAs: ColorTempPhysicalMinMiredsValue
+          constraints:
+              type: int16u
+              minValue: 0
+              maxValue: 65279
+
+    - label: "Step 1e: TH reads ColorTempPhysicalMaxMireds attribute from DUT."
+      PICS: CC.S.A400c
+      cluster: "Color Control"
+      command: "readAttribute"
+      attribute: "ColorTempPhysicalMaxMireds"
+      response:
+          saveAs: ColorTempPhysicalMaxMiredsValue
+          constraints:
+              type: int16u
+              minValue: 0
+              maxValue: 65279
+
+    - label:
+          "Step 2a: TH sends MoveToHueAndSaturation command to DUT with Hue=200,
+          Saturation=50 and TransitionTime=0 (immediately)"
+      PICS: CC.S.C06.Rsp
+      cluster: "Color Control"
+      command: "MoveToHueAndSaturation"
+      arguments:
+          values:
+              - name: "Hue"
+                value: 200
+              - name: "Saturation"
+                value: 50
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 1
+              - name: "OptionsOverride"
+                value: 1
+
+    - label: "Wait 100ms"
+      PICS: CC.S.F01
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 100
+
+    - label: "Step 2b: TH reads CurrentHue attribute from DUT"
+      PICS: CC.S.A0000
+      cluster: "Color Control"
+      command: "readAttribute"
+      attribute: "CurrentHue"
+      response:
+          constraints:
+              minValue: 170
+              maxValue: 230
+
+    - label: "Step 2b1: TH reads CurrentSaturation attribute from DUT"
+      PICS: CC.S.A0001
+      cluster: "Color Control"
+      command: "readAttribute"
+      attribute: "CurrentSaturation"
+      response:
+          constraints:
+              minValue: 42
+              maxValue: 58
+
+    - label:
+          "Step 2c: TH sends MoveToColor command to DUT, with ColorX =
+          32768/0x8000 ColorY = 19660/0x4CCC TransitionTime = 0 (immediate)"
+      PICS: CC.S.C07.Rsp
+      cluster: "Color Control"
+      command: "MoveToColor"
+      arguments:
+          values:
+              - name: "ColorX"
+                value: 32768
+              - name: "ColorY"
+                value: 19660
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 1
+              - name: "OptionsOverride"
+                value: 1
+
+    - label: "Wait 100ms"
```

Comment:
why are these waits required?

Reply:
Those were to let time to transition since some of the move to calls were asynchronous and "instant" might not always be so "instant" turns out most of them were un-necessary, will remove.